site_root/input/pages/StructureDefinition-ncpi-phenotype-intro.md

```diff
@@ -0,0 +1,4 @@
+### Scope and Usage
+The ncpi-phenotype profile, based on the Observation profile, is used to represent specific phentoypes observed to be either present or absent in a specific subject. 
```

Comment:
Catching a minor spelling error here, phentoypes should be phenotypes.

Reply:
Fixed!